src/main/resources/org/jpeek/metrics/C3.xsl

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2017-2018 Yegor Bugayenko
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
+  <xsl:param name="ctors" select="0"/>
+  <xsl:template match="skeleton">
+    <metric>
+      <xsl:apply-templates select="@*"/>
+      <title>C3</title>
+      <description>
+        <xsl:text><![CDATA[
+          C3(c) = ACMS(c) if ACMS(c) > 0 else 0; where c is a class
+          ACSM(c) = average CSM(c) for all method pairs in c (skip identity
+                    pairs, i.e. a method pairing with itself)
+          CSM(m1, m2) = cosine between LSI(m1) and LSI(m2)
+
+          LSI is a vector, defined as:
+          1) For each function, collect all unique words from variable
+          names and comments (split variable names such as `myDatabaseBook` into
+          words `my`, `database`, and `book.
+          2) Extract frequency information and build co-occurrence matrix.
+          3) Reduce matrix with SVD.
+          More info on LSI: https://en.wikipedia.org/wiki/Latent_semantic_analysis#Latent_semantic_indexing
+
+          The original paper is at /papers/marcus05.pdf.
+        ]]></xsl:text>
+      </description>
+      <xsl:apply-templates select="node()"/>
+    </metric>
+  </xsl:template>
+  <xsl:template match="class">
+    <xsl:variable name="methods" select="methods/method[($ctors=0 and @ctor='false') or $ctors=1]"/>
+    <xsl:variable name="possible_relations">
+      <xsl:for-each select="$methods">
+        <xsl:variable name="i" select="position()"/>
+        <xsl:variable name="left" select="."/>
+        <xsl:for-each select="$methods">
+          <xsl:if test="position() &gt; $i">
+            <xsl:variable name="right" select="."/>
+          </xsl:if>
+        </xsl:for-each>
+      </xsl:for-each>
+    </xsl:variable>
+    <!--
+    @todo #64:30min Implement the C3 method according to the description.
```

Comment:
@rok-povsic Can you add here, "above". So ``Implement the C3 method according to the description above. ...`` -- it is clearer that the description is in this file and should not be searched for somewhere else.